pkgs/servers/ankisyncd/default.nix

```diff
@@ -1,67 +1,59 @@
-{ lib
-, fetchFromGitHub
-, python3
-, anki
-}:
+{ lib, stdenvNoCC, fetchFromGitHub, rustPlatform
+, protobuf }:
```

Comment:
I feel like this is short enough it reads better all on one line

Reply:
I somewhat like to separate "core" dependencies to build inputs, but it doesn't really matter I guess. It's indeed short enough to fit in a line, changed.